cmd/kuberhealthy/main.go

```diff
@@ -45,12 +44,14 @@ var listenNamespace string         // namespace to listen (watch/get) `khcheck`
 var terminationGracePeriod = time.Minute * 5 // keep calibrated with kubernetes terminationGracePeriodSeconds
 
 // flags indicating that checks of specific types should be used
-var enableForceMaster bool // force master mode - for debugging
-var enableDebug bool       // enable debug logging
-var logLevel = "info"
+var DSPauseContainerImageOverride string // specify an alternate location for the DSC pause container - see #114
+// DSTolerationOverride specifies an alternate list of taints to tolerate - see #178
+var DSTolerationOverride []string
```

Comment:
hmm can you check is this is being used in the kuberhealthy project anywhere? pretty sure this was for the daemonset check and thats now in a different package!